hello.go

```diff
@@ -10,5 +10,7 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	http.ListenAndServe(":8888", http.HandlerFunc(helloHandler))
+	router := http.NewServeMux()
+	router.HandleFunc("/hell", http.HandlerFunc(helloHandler))
```

Comment:
The handler pattern should be `/hello`, not `/hell` 👹. Also, please fix the typo in the commit title.

Reply:
Fixed the title and the typo in the handler path in 49da4e5.